libs/blocks/aside/aside.css

```diff
@@ -450,93 +469,104 @@
     padding: 0;
   }
 
-  .aside.split.half .background img {
+  .aside.split.half .split-image img {
     width: 50vw;
     max-width: 1396px;
   }
 
-  .aside.split.half .foreground.container .text {
-    flex: 0 0 41.72%;
+  .aside.inline .foreground.container .text {
+    flex: 0 0 calc(60% - var(--spacing-s));
+    max-width: none;
   }
 
-  .aside.split.right .foreground.container {
-    flex-direction: row-reverse;
+  .aside.notification .foreground.container .text {
+    padding-right: 0;
   }
 
-  .aside.split.right .background img {
-    left: 0;
+  .aside.notification.small .foreground.container .text {
+    flex-flow: row nowrap;
+    align-items: center;
+    flex-grow: 1;
   }
 
-  .aside.augmented-image .background {
-    clip-path: inset(19% 0);
+  .aside.notification.small .foreground.container .text .body-M {
+    margin: 0;
   }
 
-  .aside.augmented-image .foreground.container .image {
-    flex: 0 0 57%;
-    max-width: none;
+  .aside.notification.small .foreground.container .text .body-M.action-area {
+    margin-left: 24px;
+  }
+
+  .aside.split.half .foreground.container .text {
+    flex: 0 0 41.72%;
   }
 
-  .aside.augmented-image .foreground.container .text {
-    flex: 0 0 43%;
+  .aside.right .foreground.container {
+    flex-direction: row-reverse;
```

Comment:
Silly question. For this new aside right variant. Why wouldn't you just author the text to the right and the image to the left (opposite of default authoring)?

Like this:
![image](https://user-images.githubusercontent.com/2539954/219518794-4c5039cc-d66d-4795-975d-18342c3b08fd.png)

I don't think we need this right variant. Authors should author it that way.

Reply:
oops this is a reply to the other question, not sure why it shows here @rgclayton

Reply:
Thanks! That helps.

Reply:
@rgclayton okay the right alignment is now coming from the authored layout instead of a variant class.